lib/utils.js

```diff
@@ -34,27 +34,27 @@ function relayEvents(listener, emitter, events) {
   events.forEach(eventName => listener.on(eventName, event => emitter.emit(eventName, event)));
 }
 
-// Grab Kerberos values if they exist, otherwise set them to null
-let Kerberos = null;
-let MongoAuthProcess = null;
+function retrieveKerberos() {
+  let kerberos;
 
-try {
-  const kerberos = requireOptional('kerberos');
-  if (kerberos) {
-    Kerberos = kerberos.Kerberos;
-    MongoAuthProcess = kerberos.processes.MongoAuthProcess;
+  try {
+    kerberos = requireOptional('kerberos');
+  } catch (err) {
+    if (err.code !== 'MODULE_NOT_FOUND') {
+      throw new Error('The `kerberos` module was not found. Please install it any try again.');
```

Comment:
Is this throwing an error for `kerberos not found` for errors not related to `MODULE_NOT_FOUND`?

Reply:
oops, good catch!